server/auth/auth_handler_factory.test.ts

```diff
@@ -120,19 +123,6 @@ describe('test authentication factory', () => {
     expect(auth.type).toEqual('basicauth');
   });
 
-  test('get basic auth with empty auth type: string', async () => {
```

Comment:
Why was this test removed?

Reply:
This PR looks good to me once this comment is addressed.

Reply:
seems like a duplicate of a test above this one